config.py

```diff
@@ -1 +1,17 @@
 ROTATE_MAX_CHARS = 15
+
+SUBGROUPS = {'EE': ['DiddleDad', 'fgsrfug', 'Selfishshellfish', 'aaronschraner'],
+             'CS': ['tolvstaa', 'Codification', 'codysseus'],
+             'CH': ['TheChemE'],
+             'BO': ['adny_bot'],
+             'AN': ['adny_bot', 'TheChemE', 'tolvstaa'],
+             'CO': ['Codification', 'codysseys'],
+             'GR': ['tolvstaa', 'Codification'],
+             'NE': ['irandms'],
+             'AL': []}
+
+for users in SUBGROUPS.values():
+    SUBGROUPS['AL'] += users
+
+SUBGROUPS['AL'] = list(set(SUBGROUPS['AL']))
```

Comment:
What is the necessity of this line vs Lines 13/14 together?

Reply:
There isn't any. It's supposed to say 'EV' on the left, for the case that someone wants to say 'everyone'